mysql/infoschema_test.go

```diff
@@ -226,6 +273,10 @@ func TestProcessSQLData_MultiCol(t *testing.T) {
 			args:  []driver.Value{"test", "test"},
 			cols:  []string{"column_name", "constraint_type"},
 			rows:  [][]driver.Value{}, // No primary key --> force generation of synthetic key.
+		}, {
+			query: "SELECT (.+) FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS (.+)",
+			args:  []driver.Value{"test", "test"},
+			cols:  []string{"REFERENCED_TABLE_NAME", "COLUMN_NAME", "REFERENCED_COLUMN_NAME", "ordinal_position"},
```

Comment:
"ordinal_position" -> "ORDINAL_POSITION"?

Reply:
yes, i missed to remove it from this test, removed it.